rediscluster/client.py

```diff
@@ -86,6 +90,11 @@ class StrictRedisCluster(StrictRedis):
         string_keys_to_dict([
             "SSCAN", "HSCAN", "ZSCAN", "RANDOMKEY",
         ], first_key),
+        {
```

Comment:
Please be consitent with all other defentiions in this variable. Even if it is only one key to method you should use `string_keys_to_dict` to define all entries in RESULT_CALLBACKS